cosmogony/src/zone.rs

```diff
@@ -49,6 +49,8 @@ pub struct Zone {
     pub admin_level: Option<u32>,
     pub zone_type: Option<ZoneType>,
     pub name: String,
+    pub loc_name: Option<String>, // add them in full labels ??
```

Comment:
I think this comment was intended as a reflection before opening https://github.com/hove-io/mimirsbrunn/pull/752?